bin/test-browser.js

```diff
@@ -87,6 +89,34 @@ class RemoteRunner {
 
       handlers[event.name].forEach(function (handler) {
         handler(obj, event.err);
+
+        if (event.err && stackConsumer) {
+          let stackMapped;
+          const mappedStack = require('stacktrace-parser')
```

Comment:
can we move require statements to top of file? this makes future conversion to esm much less painful and simpler to see what is going on in a module...